code/API_definitions/Traffic_Influence.yaml

```diff
@@ -79,18 +82,24 @@ info:
     by a source port and a destination port and protocol. To optimise the flow
     from more source ports or destination ports or protocols, the TI API must be
     invoked many times.\
+    It is important to notice that it is a task of the Service Application to
+    use the most adequate means to send the application flow to the right
+    server (EAS), for example using Edge DNS or using the target EAS IP address.
+    The TI API provides back the selected EAS instance identifier from which the
+    Service Application can retrive the destination IP address of its backend
+    server, to properly configure the service data flow.\
```

Comment:
it is a good proposal. In my understanding one API is : Edge-Application-Management. Is my understanding correct?

Reply:
Yes i think GET /apps/{appId}/instances: from Edge-Application-Management. seems to be an option and possibly even  the Application_Endpoint_Discovery_API could also be used in some of the scenarios though it will work without any instance identifier.